process_manager/views/partials.py

```diff
@@ -14,6 +17,33 @@
 from ..tables import ProcessTable
 
 
+def handle_errors(
+    view_func: Callable[[HttpRequest], HttpResponse],
+) -> Callable[[HttpRequest], HttpResponse]:
+    """Decorator to handle errors.
+
+    Args:
+        view_func: The view function to be wrapped.
+
+    Returns:
+        The wrapped view function.
+    """
+    logger = logging.getLogger("django")
+
+    def wrapped_view(request, *args, **kwargs) -> HttpResponse:  # type: ignore
+        try:
+            return view_func(request, *args, **kwargs)
+        except Exception as e:
+            logger.exception(e)
+            return render(
+                request,
+                "process_manager/partials/error_message.html",
+                {"error_message": "An error occurred while processing your request."},
```

Comment:
The template doesn't render the `error_message` variable so this can be removed.